docs/advanced/verifying-binaries.md

```diff
@@ -1,53 +1,325 @@
+   Download the `manifest.txt` and the signature `manifest.txt.asc`. Use GnuPG
+   to check the signature against the public Decred release signing key.
+
+     * SUCCESS: If the signatures match, you know you can trust that the
+       manifest came directly from the Decred project, and that the hashes
+       inside it are trustworthy.
+
+     * FAIL: If the signatures don't match, then your manifest is not the
+       official Decred project manifest. Do not continue, delete the files you've
+       downloaded, and check for security problems before trying again.
+
+2. **Verify that the binary hash matches the manifest hash:** Hash your
+   downloaded binary and verify that its hash matches the hash in the Decred
+   project's `manifest.txt`.
+
+     * SUCCESS: If your hash matches the hash in the manifest, you know you can
+       trust and run the binary.
+
+     * FAIL: If your hash doesn't match, your binary doesn't match the official
+       Decred project's binary release, and you should not continue. Delete the
+       binary you've downloaded, and check for security problems before trying
+       again.
 
 ---
+## Detailed Instructions
+
+### Preliminary Steps
+
+First, go to the [Decred releases](https://github.com/decred/decred-binaries/releases)
+page and download:
+
+* The binary installer for your specific OS / architecture
+* The file manifest and hashes, ending in `-manifest.txt`
+* The signature for the manifest, ending in `-manifest.txt.asc`
+
+Also, make sure GnuPG is installed:
+
+* On Windows, install GnuPG with the latest version of [gpg4win](https://www.gpg4win.org/).
+* On macOS, install GnuPG with the latest version of [GPGTools](https://gpgtools.org/).
+* On Linux, GnuPG is installed by default.
+
+With GnuPG installed, the following instructions will work from any terminal on
+Windows, macOS, or Linux/UNIX
+
+### Add the Decred project's release public keys to your GnuPG keyring.
+
+You only need to do this step one time, so you can skip this step when
+verifying later releases on the same computer.
 
-The official
-[Decred releases](https://github.com/decred/decred-release) have
-signed hashes that you should check to confirm that they are the same
-binaries that the developers posted.  This page provides instructions
-for that.
+There are several ways to import the Decred release public keys into GnuPG.
+The most direct method is to use the ASCII Key Block below:
 
-In order to verify binaries or other signed files from the Decred
-Project, there are a couple pieces of software required:
+??? info "The Decred Release ASCII Key Block (click to expand)"
 
-* SHA-256 -- Once you download your file(s), you need to check their
-  SHA-256 hashes, so you may need to download a tool to do this,
-  depending on your OS.
-* GnuPG or PGP -- This is required to import public keys and verify
-  signatures. Examples below use GnuPG.
+        -----BEGIN PGP PUBLIC KEY BLOCK-----
```

Comment:
please make this a code section (put three backticks on a line before and following this key section) and remove the leading whitespace before each line in the key.  The key copied as-is from this file doesn't work for verification, at least with the tools I am using.

Reply:
This does not match what I have locally. Here is the diff output showing where they diverge:

```
$ diff -uNp old.pub new.pub
--- old.pub     2020-05-07 13:16:55.159845300 -0500
+++ new.pub     2020-05-07 13:17:20.720221400 -0500
@@ -23,74 +23,62 @@ Q3IHxZQKaCcuzfy65q48QXj9AMX1KSPYZqze51wu
 CjsLxL7MqCR7zHxfmzi7oyNHtqMPdoG4MPFfamoSHgiN1Xck1OKtaVstq0VAmZCp
 ixl+e327jwYnF73QtZ7TWrJj6UO1chnGGQzVE1JtHCqzbaVbRVg/8gYClG5aUAjP
 99pOv1QquwixuEArcTF91XNjhQYNuOitgSuqCC9b6fCpyXRzG7EB+z86W4J+rwF5
-7ATyPKE/rzngiRW0i9KFot5dBFZzyljtPaCJAjgEEwECACIFAlapILECGwMGCwkI
-BwMCBhUIAgkKCwQWAgMBAh4BAheAAAoJEG32NKp2CK8EilQP/0lobzbxfNbQCI++
-QGKSwTcy3mkYzIqKWujZQAwVoQk5W2J5cKft1kcS6exFCyj+DeBrWMVPXVQ+YVjC
-ODpL9Ewczz0aLOQihFKn5NgJ2epy5+BrpniBH7Frt9v/FVtc0+azhIHg3flvbY9S
-KwvNNbnNRI5DwHcBqySs+4m11qtGbVwgz2OdGHAL2XU8aijn3Y38XWzJpJ14xEBl
-jOXg7vgIuH7cYWi4S754hsnwb5iS9sN41lXX3D2wwQ/FtuNXDB/EpDJDDi+0vsS6
-sk5Rpe3gyT33aG5Vqk1aXk1yI/JRtgJzhXX9CHZ7CGR8ki5Ri7iUXfwGWgA6JARE
-8xcKW1gu4CJvvIZGe+8SKeX2g/Xcw4GMaCV4V9ReMiuYCXOOmyg0zwQ/OjJMOxYA
-UljYzUs7HIEyqN2u4adPlQhPYgTEYyRFIzsW7dvmmL+YKilT6cKp3GpX2dJiI3kE
-AX0d2aKsLpQFNLoA36BqCIrcbXbrpap1HFnFzx9F10blzL57dv8AmU48TShuyQ5T
-JeCuILJ7ZYxvcxnjFKYU8Wwwz/L52H0vOdx7gS7lhxL9HMoil9bfWZLu2i1TGGxx
-lsKf/QIYW5YzQXsGFieVv95VcZQpdRN/a3yQTwlePftnfQ2eZ0JARAP1IAcqnr64
-0aKnLs3WzuI5gBTH2P2B9ix6DDD7uQINBFapILEBEAC4BpeMb7QNk6mfKk2nrkDF
-dj2UgigAw2xsnkEUpHG6IufSlTOHF2hiJP7k86lbrIZGfQM9+9WBb2m+kik1Zh2I
-53vvXD460ZtGBBzD7UMvAf5BOvrpnX7rmqtjLpGUvPhrQ/6h8LrBH29jCn/8C8yL
-fl3B7A11C3YaxRVmR2TBXjMaYpmJ6Qhho4Jbw36/qscnZcPbFKTOs70uYAZD5hT7
-PYBQs+496bLiDTjk5SkU2WsSRQG7+IDmTIMC0tzIPKcf1H14lXDAqSWDJsEuF/6g
-zPc+Kg3GL0KW0hPHDb5+z5pfzKKBJQxWBwaPjAbGsn/WKFmuLMR6NzXZLoSt/EQq
-Bd6Ud9goW+4JKeZhlVEWIZ/C+uNOr5eqEM1qiaEJW3Hrw5lxn4PEYZ2h59VPjM46
-jsn5baRx54Wo/4oX8DpDlTyPM8ZOVkXDhHVgkHagygwLkiCQIyH9/htdVyIf0+33
-zBS+oIsW8TJgbMaVrb9zy8BFcKfpICIjm5gCdxAb5xtO0pJSiSv4Ga0TTPwhzbHF
-f6JpCohrg0ZpTpd2b4ZNOyshpWU8b9tdbSaoP3CQnpa7erzxOSI+xkAJODG+7DAz
-qZrs0dMSAzlTG8gKmq5ceVWFA68gwDOZk6ObV4qcLaAakTYMWPzDMYjVD+NiNVJx
-OYTBsj9lo8LaJSojNUmXzwARAQABiQIfBBgBAgAJBQJWqSCxAhsMAAoJEG32NKp2
-CK8EKJwQAJAKqVuurLX2ApEgeLUVqb18s7kKmDC9MBy11zhmAzH51xrJimzg4j3v
-QUjZqmV4iN4wPti/ME5RhwSgE9PeDXupsmGf//pD0YmTIWvOMhj4hASc4l6uNhlo
-E2j5tN0A8IZBVQO1PvJdVYi6KJIYZy07qOg79qYQR4yYAXDLZQTlyBefvhVbk0H9
-Ds8cC8gH9ag6Yn9t4TCfGhx7NP2j9W29OtnuDFt6GssgUt/1o1WILdMn2DzAdNr7
-f6VDCSLKMjc3WQFe1XmrbR/xiH2SqKAOF6UIx++H4p7XPZyBmDcdbGzkputPYey0
-tsvEN3ndyNLBsTgzPLALKiiXxvts798fjFWnZFVq1KmcZMj4+4yJLIBTBU1ZW0cC
-F3e31qzAEDJmrKcwhN9IzVWAhRhHxpkKc2oADR5Lmq9CcXMOYZF9aS83YlIJKZ8U
-WEgna802dzBcckBt2RvYxDYqs4iLLDdHjMGahTM3uieofIUXApUuSAMfTFvq5HSq
-0/UU/UekE+NMUn3UtW2XLl9B49aB5bcUtOYtcbIJu8lhHuNXc2+zL/s9zCTsVx5P
-XdCmEE81HfSxU/+7yhSxs10ixIporA2OxLYTUkzDDrqlK2N0ENjQp+39eVNTkGJd
-15SnkvhqDukr11hTqdIgrwqrHTF6o2mVK79h2AqZqbi5ISCPc/MvuQINBFapIcYB
-EACUE78/3F2br1jVCD8w/MC6rfxkleKjdfsafkkI10UPzhhMZAhgX1sXehF8luKC
-sbWJ/d92j5dHOy5O8j6WuUfVWZgHQh3HqTBujz3lMYZXC3JCsUPajpQx5VH43JRj
-pOz/Pw5Vy9RIS3UJ78vIAC2jqPc0wVknZmQ5JnF6nNGyU0AJYX7kwBM4685avPsI
-tdpdix93Z3NEcqmS1B4PF3bCU96gHzAAw7IfCuF6TGzraHV4OPVJxa8GJp4Ziwnn
-KD4vZzMx8FYMd8Egw50nsjFDL/DN2cFU512k/RSFFzXJ9y+NwnFwtZ3EobO8kQU0
-fFqW4AFyfwsJKBAl8JlhgqzqeepRp1r0y7xmuxKjLxPgnps6ucSYbsqQhBztUCy0
-fNW80gdb4kWz9bavY9165ALwCNuGQgLcLziKg4SwKiTJqurFcnAAgBg9X3pBFKJQ
-seiarGELGaf/ptVA74auYNeqySJoSRpUe+/9WvX05hcvb786/KpOBBfYPtmWW2tZ
-abo1T2FGiECNuqh0BqyxVQ3IVSBAY1IQjIGoXqR1Vh5kxmqW+5HxdlCKRZDBYk/5
-ufT2d1GnLPkc+a8dm0PZhfzkM2fYblIpLEwd2w/xZZhmyYnDIkoFZxEu7RzT3t3y
-ackos12ymyP4/qsCDf76VJ7KJhAE14MPiUHK0jGfalEZPQARAQABiQQ+BBgBAgAJ
-BQJWqSHGAhsCAikJEG32NKp2CK8EwV0gBBkBAgAGBQJWqSHGAAoJEG2Jft9RigMd
-GqEP/iIB3E2JpzlKAVBkBu0kQU7CHX7P4zcACayE3buOfzjgzLVk6IdwboH/LYT2
-0w+Qwkqo1MV6uTe+831Hd9jRLyEuyxklGliYbXvdGbA+vtpdYcRiVnR61ATUg3Yu
-d8MoLsqw+IK61W9e1M2puElKQ6Px/UmJTnfm3OsAnZ2BGJEpYJS1IYkxAKXqMVPE
-bdZlMD+8/O9Aq0h5ySW0aIn6GNiUbzPzq9QMviMHR7Nolnw4aangtDUAmlqHO6Gh
-3mUaIbPjt19HOFwHnCnv11CZRbyyoXonhZFxOATS13Av+hGg6J8S/gGSiV9fT+53
-O1BodBygKJ8Y7JJDFIc/rTt05HaqHbNhucFqCUf5WuDOQabjiWWkgxjUdh050CzK
-reYn8TGkTd6OCcmedUB3J4HmbhmwpkBw3ybAWvYhPoFi92/P7FtSThkaSbydnD8V
-Men8wy+OuhJtvDnxLc4YGlxWqsW9WelsdxAZZNdSMPDrpmvkXJflmvCiP+4wehvx
-Z3R2cgzOZjjYZlrD98IMjNdo87YTs4pxi+mEQNLYrLR404ZbhgTNkdFjKByM6EOl
-F5xebblJjaTSLvaCs5p1lLdLTMSD3+1JUMLYRBuLN46ePLlxuDwtgum5hiNes2Ry
-DGa/Gln73b/YtuGigcqi7ouesHuhcY0OwFuB4pu/8r+MbceYj3gP/jmZbVn2Yp3Q
-qCgQYf/XFiZ5cgXitanBeYPa4A9WNpK2CuF0mtcwaE/vhTki37N9y5OQjpllJ/fL
-equFTw7IPHcqcrQ7Fgeaf/lrWgVyNOUWiIJ9OJA4bjAJEMoD+qut8Ci8SPHe14Fr
-94xP+ZrM7b9ZmEPvQTAQWip3Gtx3Ydv8U5z4eCeNPU6PnOxEEDvlBsyQp2wG+8ft
-pLaidPZgfmdYoqLFIxhQiMUbQiDEmdZdlXCvMdOGF5oCxJDVFTovlaA7VgJMiUJG
-hO8TVU+hsH6IeUMoOzCucKN6jlbaYTH9gOm0eclp+cE2BRSPYCh+B9J7uJm7uZR6
-mW6uTx+pSyot6/eaJDo/EyBzvYdJK6eIdllFUfAO/S1LD160RPIsnevoYo7Yce3B
-Ud2Pasrf+8Ptp/sFHT1hwMCz44SnkK4un9VQ4L0WsMnNQZKGmyyE1+Ydz6iHwKHf
-lkGoJOmBU269MNKAh4qmwW9uG+T6Jxc50GLSzBAETtx3MYznEYjkcjCLnlb+x7Yh
-HBRWMXiXCQ5atlWig2t/urrkrkSegVNnaP0fOSL71qihX0sRlsvEJCIWktKWVmzt
-gcn75IpFzbTVTMtZoOgdFUSDPRSqNHA8hL+Itz2dUqHSktQ7pFK6ogIkvzx+7M7n
-O6GuapL0x1I9SeIk9WX9VRrpZmENhlri
-=JrM6
+7ATyPKE/rzngiRW0i9KFot5dBFZzyljtPaC5Ag0EVqkhxgEQAJQTvz/cXZuvWNUI
+PzD8wLqt/GSV4qN1+xp+SQjXRQ/OGExkCGBfWxd6EXyW4oKxtYn933aPl0c7Lk7y
+Ppa5R9VZmAdCHcepMG6PPeUxhlcLckKxQ9qOlDHlUfjclGOk7P8/DlXL1EhLdQnv
+y8gALaOo9zTBWSdmZDkmcXqc0bJTQAlhfuTAEzjrzlq8+wi12l2LH3dnc0RyqZLU
+Hg8XdsJT3qAfMADDsh8K4XpMbOtodXg49UnFrwYmnhmLCecoPi9nMzHwVgx3wSDD
+nSeyMUMv8M3ZwVTnXaT9FIUXNcn3L43CcXC1ncShs7yRBTR8WpbgAXJ/CwkoECXw
+mWGCrOp56lGnWvTLvGa7EqMvE+Cemzq5xJhuypCEHO1QLLR81bzSB1viRbP1tq9j
+3XrkAvAI24ZCAtwvOIqDhLAqJMmq6sVycACAGD1fekEUolCx6JqsYQsZp/+m1UDv
+hq5g16rJImhJGlR77/1a9fTmFy9vvzr8qk4EF9g+2ZZba1lpujVPYUaIQI26qHQG
+rLFVDchVIEBjUhCMgahepHVWHmTGapb7kfF2UIpFkMFiT/m59PZ3Uacs+Rz5rx2b
+Q9mF/OQzZ9huUiksTB3bD/FlmGbJicMiSgVnES7tHNPe3fJpySizXbKbI/j+qwIN
+/vpUnsomEATXgw+JQcrSMZ9qURk9ABEBAAGJBD4EGAECAAkFAlapIcYCGwICKQkQ
+bfY0qnYIrwTBXSAEGQECAAYFAlapIcYACgkQbYl+31GKAx0aoQ/+IgHcTYmnOUoB
+UGQG7SRBTsIdfs/jNwAJrITdu45/OODMtWToh3Bugf8thPbTD5DCSqjUxXq5N77z
+fUd32NEvIS7LGSUaWJhte90ZsD6+2l1hxGJWdHrUBNSDdi53wyguyrD4grrVb17U
+zam4SUpDo/H9SYlOd+bc6wCdnYEYkSlglLUhiTEApeoxU8Rt1mUwP7z870CrSHnJ
+JbRoifoY2JRvM/Or1Ay+IwdHs2iWfDhpqeC0NQCaWoc7oaHeZRohs+O3X0c4XAec
+Ke/XUJlFvLKheieFkXE4BNLXcC/6EaDonxL+AZKJX19P7nc7UGh0HKAonxjskkMU
+hz+tO3Tkdqods2G5wWoJR/la4M5BpuOJZaSDGNR2HTnQLMqt5ifxMaRN3o4JyZ51
+QHcngeZuGbCmQHDfJsBa9iE+gWL3b8/sW1JOGRpJvJ2cPxUx6fzDL466Em28OfEt
+zhgaXFaqxb1Z6Wx3EBlk11Iw8Ouma+Rcl+Wa8KI/7jB6G/FndHZyDM5mONhmWsP3
+wgyM12jzthOzinGL6YRA0tistHjThluGBM2R0WMoHIzoQ6UXnF5tuUmNpNIu9oKz
+mnWUt0tMxIPf7UlQwthEG4s3jp48uXG4PC2C6bmGI16zZHIMZr8aWfvdv9i24aKB
+yqLui56we6FxjQ7AW4Him7/yv4xtx5iPeA/+OZltWfZindCoKBBh/9cWJnlyBeK1
+qcF5g9rgD1Y2krYK4XSa1zBoT++FOSLfs33Lk5COmWUn98t6q4VPDsg8dypytDsW
+B5p/+WtaBXI05RaIgn04kDhuMAkQygP6q63wKLxI8d7XgWv3jE/5msztv1mYQ+9B
+MBBaKnca3Hdh2/xTnPh4J409To+c7EQQO+UGzJCnbAb7x+2ktqJ09mB+Z1iiosUj
+GFCIxRtCIMSZ1l2VcK8x04YXmgLEkNUVOi+VoDtWAkyJQkaE7xNVT6Gwfoh5Qyg7
+MK5wo3qOVtphMf2A6bR5yWn5wTYFFI9gKH4H0nu4mbu5lHqZbq5PH6lLKi3r95ok
+Oj8TIHO9h0krp4h2WUVR8A79LUsPXrRE8iyd6+hijthx7cFR3Y9qyt/7w+2n+wUd
+PWHAwLPjhKeQri6f1VDgvRawyc1BkoabLITX5h3PqIfAod+WQagk6YFTbr0w0oCH
+iqbBb24b5PonFznQYtLMEARO3HcxjOcRiORyMIueVv7HtiEcFFYxeJcJDlq2VaKD
+a3+6uuSuRJ6BU2do/R85IvvWqKFfSxGWy8QkIhaS0pZWbO2ByfvkikXNtNVMy1mg
+6B0VRIM9FKo0cDyEv4i3PZ1SodKS1DukUrqiAiS/PH7szuc7oa5qkvTHUj1J4iT1
+Zf1VGulmYQ2GWuK5Ag0EVqkgsQEQALgGl4xvtA2TqZ8qTaeuQMV2PZSCKADDbGye
+QRSkcboi59KVM4cXaGIk/uTzqVushkZ9Az371YFvab6SKTVmHYjne+9cPjrRm0YE
+HMPtQy8B/kE6+umdfuuaq2MukZS8+GtD/qHwusEfb2MKf/wLzIt+XcHsDXULdhrF
+FWZHZMFeMxpimYnpCGGjglvDfr+qxydlw9sUpM6zvS5gBkPmFPs9gFCz7j3psuIN
+OOTlKRTZaxJFAbv4gOZMgwLS3Mg8px/UfXiVcMCpJYMmwS4X/qDM9z4qDcYvQpbS
+E8cNvn7Pml/MooElDFYHBo+MBsayf9YoWa4sxHo3NdkuhK38RCoF3pR32Chb7gkp
+5mGVURYhn8L6406vl6oQzWqJoQlbcevDmXGfg8RhnaHn1U+MzjqOyfltpHHnhaj/
+ihfwOkOVPI8zxk5WRcOEdWCQdqDKDAuSIJAjIf3+G11XIh/T7ffMFL6gixbxMmBs
+xpWtv3PLwEVwp+kgIiObmAJ3EBvnG07SklKJK/gZrRNM/CHNscV/omkKiGuDRmlO
+l3Zvhk07KyGlZTxv211tJqg/cJCelrt6vPE5Ij7GQAk4Mb7sMDOpmuzR0xIDOVMb
+yAqarlx5VYUDryDAM5mTo5tXipwtoBqRNgxY/MMxiNUP42I1UnE5hMGyP2Wjwtol
+KiM1SZfPABEBAAGJAh8EGAECAAkFAlapILECGwwACgkQbfY0qnYIrwQonBAAkAqp
+W66stfYCkSB4tRWpvXyzuQqYML0wHLXXOGYDMfnXGsmKbODiPe9BSNmqZXiI3jA+
+2L8wTlGHBKAT094Ne6myYZ//+kPRiZMha84yGPiEBJziXq42GWgTaPm03QDwhkFV
+A7U+8l1ViLookhhnLTuo6Dv2phBHjJgBcMtlBOXIF5++FVuTQf0OzxwLyAf1qDpi
+f23hMJ8aHHs0/aP1bb062e4MW3oayyBS3/WjVYgt0yfYPMB02vt/pUMJIsoyNzdZ
+AV7VeattH/GIfZKooA4XpQjH74fintc9nIGYNx1sbOSm609h7LS2y8Q3ed3I0sGx
+ODM8sAsqKJfG+2zv3x+MVadkVWrUqZxkyPj7jIksgFMFTVlbRwIXd7fWrMAQMmas
+pzCE30jNVYCFGEfGmQpzagANHkuar0Jxcw5hkX1pLzdiUgkpnxRYSCdrzTZ3MFxy
+QG3ZG9jENiqziIssN0eMwZqFMze6J6h8hRcClS5IAx9MW+rkdKrT9RT9R6QT40xS
+fdS1bZcuX0Hj1oHltxS05i1xsgm7yWEe41dzb7Mv+z3MJOxXHk9d0KYQTzUd9LFT
+/7vKFLGzXSLEimisDY7EthNSTMMOuqUrY3QQ2NCn7f15U1OQYl3XlKeS+GoO6SvX
+WFOp0iCvCqsdMXqjaZUrv2HYCpmpuLkhII9z8y8=
+=+HJT
 -----END PGP PUBLIC KEY BLOCK-----
```

Reply:
I just tried it myself, and it works for me. I think I know what the problem is.

In order to put the key in a hidden dropdown, I can either use HTML or the Mkdocs plugin.

* If I use HTML, I can use \<details>\<summary>, followed by the \``` code block.
    * This allows it to be copied correctly from the raw text because there are no leading or trailing spaces on each line.
* If I use the Mkdocs plugin, I have to use \??? info "title", followed by indenting every line. The \``` code block no longer works because each line is indented, so if I want to use a code block, I need to indent each line another 4 spaces.
    * This doesn't allow the key to be copied directly from the raw text because of all the leading spaces on each line.

Finally, either option works when built and hosted using Mkdocs, but only the HTML version works with the raw text file. @jholdstock, are you OK with me switching to HTML so that the raw text file doesn't introduce future problems?

Reply:
To clarify, the difference I'm seeing is not about the spacing.  I adjusted that already.  Rather, the data is not the same after a certain point.  The one being introduced here is `...PaC5a...` versus the one I have locally which is `...PaCJA...`.

Reply:
Ok, using your terminology `old` and `new` from your diff:

* Result of importing `old`:
```
gpg: key 6DF634AA7608AF04: public key "Decred Release <release@decred.org>" imported
```
```
pub   rsa4096 2016-01-27 [SC]
      FD13B6835E248FAF4BD1838D6DF634AA7608AF04
uid           [ unknown] Decred Release <release@decred.org>
sub   rsa4096 2016-01-27 [E]
sub   rsa4096 2016-01-27 [S]
```

* Result of importing `new`:
```
gpg: key 6DF634AA7608AF04: public key "Decred Release <release@decred.org>" imported
```
```
pub   rsa4096 2016-01-27 [SC]
      FD13B6835E248FAF4BD1838D6DF634AA7608AF04
uid           [ unknown] Decred Release <release@decred.org>
sub   rsa4096 2016-01-27 [S]
sub   rsa4096 2016-01-27 [E]
```

I don't see any differences.

Reply:
`$ gpg --export --armor release@decred.org > existing.txt`

```
-----BEGIN PGP PUBLIC KEY BLOCK-----

mQINBFapILEBEADZxw+4Z8LlqsXCz3j3Ap04SF8zYenlsw123OJZEh9RFERd19bo
+l2RueFqi5vJDGWpXZ+eHxvgevvOO3r0AiIgAByAP7RQQxip4j6M2xnEBdVb9UV5
baO93JcyBRDnII/zh6Zf4pqngiYEz7juySsnVMrE7IFmIdT/WfoGW6FX8/kRXyzf
RTScPZKxIEqwHSlLftlVGSxKL9H+RumEUjPaazLvER1XxtfvcaMGLpatZV3ccqjX
3O+b3plccx0KbMStMtsB0VI+kcaFKg2gIQrbkHKzpDUI2AdaNJJCodM6j3LphBSS
5ZXOknyThpYsxDDyYcncWC9gXrGJfrirO/DPrV1NIj4luBbwyWVT1x9rp2PcUYmG
ZIq0cR4C/mxtlo9OKoyj2cxgoT4WlzlCimRSGtylkWOAx6JQLeKPWt1tZquJB3NT
Jby7x62AyqXhSMnNPDROKL37tkyWehFlAm8KNa6P8R4vctjjJDQ61yw6jskkJaNA
Qz2UNAX+Ztx5KA0Z2HEmJb1jp67EH+3kfAv7R1U51gutzuM7J+vDnNQbwQeuq6os
Y/yssU+OQidLjkojZc7aHz2iym6cw6IlrLTLCnnQQPzAe8CjskrfjwDOejDkPCYO
AkMtgs6/rsJZnCFJ8Pro7NbREt5KT06CPp4nqXNRbtBOHsa1n8wb/M9TQwARAQAB
tCNEZWNyZWQgUmVsZWFzZSA8cmVsZWFzZUBkZWNyZWQub3JnPokCMgQTAQIAHAIb
AwIeAQIXgAUCVqkhhwYLCQgHAwIFFQoJCAsACgkQbfY0qnYIrwRtvA/+JAWw/8cU
xNe5vyWle4uzHakyO25qdH4+TonHbhqyoF2F8BLvkOU3CmtBgXRAZ8Z2jdAczfuJ
u1338BJuHoAIVpvtPzRLLsrrl3LOruiCCYsxm7FKpdYWGanTwpUaHiqHj5LaeIt7
IQjPT3g+uIZ6NsN2RZDzjXZOFD0kZ9EM2b0GqrNpuIQTJafaqGSkOohPiA6b+Sen
7E/XriEo2RWHgNJP7m4xKF0nGDdMxmV0Wrcv6PBJLhZF1RMZSSsFFeTkoHti3113
H9oTKmuw5TUIfYjenGY2rXzkR8xZmCr6BiiUgRFVyVtToG6skLtUvkN3aT0QueDt
u+Lr7QFpM3T9cYqJsg4Gd/9gPUPU6o6r82YlOmB7AQuu99pZ/4KrNIY8saZPRNuS
Q3IHxZQKaCcuzfy65q48QXj9AMX1KSPYZqze51wu8iywfOsq+GC1zw8+gI6alDUl
CjsLxL7MqCR7zHxfmzi7oyNHtqMPdoG4MPFfamoSHgiN1Xck1OKtaVstq0VAmZCp
ixl+e327jwYnF73QtZ7TWrJj6UO1chnGGQzVE1JtHCqzbaVbRVg/8gYClG5aUAjP
99pOv1QquwixuEArcTF91XNjhQYNuOitgSuqCC9b6fCpyXRzG7EB+z86W4J+rwF5
7ATyPKE/rzngiRW0i9KFot5dBFZzyljtPaCJAjgEEwECACIFAlapILECGwMGCwkI
BwMCBhUIAgkKCwQWAgMBAh4BAheAAAoJEG32NKp2CK8EilQP/0lobzbxfNbQCI++
QGKSwTcy3mkYzIqKWujZQAwVoQk5W2J5cKft1kcS6exFCyj+DeBrWMVPXVQ+YVjC
ODpL9Ewczz0aLOQihFKn5NgJ2epy5+BrpniBH7Frt9v/FVtc0+azhIHg3flvbY9S
KwvNNbnNRI5DwHcBqySs+4m11qtGbVwgz2OdGHAL2XU8aijn3Y38XWzJpJ14xEBl
jOXg7vgIuH7cYWi4S754hsnwb5iS9sN41lXX3D2wwQ/FtuNXDB/EpDJDDi+0vsS6
sk5Rpe3gyT33aG5Vqk1aXk1yI/JRtgJzhXX9CHZ7CGR8ki5Ri7iUXfwGWgA6JARE
8xcKW1gu4CJvvIZGe+8SKeX2g/Xcw4GMaCV4V9ReMiuYCXOOmyg0zwQ/OjJMOxYA
UljYzUs7HIEyqN2u4adPlQhPYgTEYyRFIzsW7dvmmL+YKilT6cKp3GpX2dJiI3kE
AX0d2aKsLpQFNLoA36BqCIrcbXbrpap1HFnFzx9F10blzL57dv8AmU48TShuyQ5T
JeCuILJ7ZYxvcxnjFKYU8Wwwz/L52H0vOdx7gS7lhxL9HMoil9bfWZLu2i1TGGxx
lsKf/QIYW5YzQXsGFieVv95VcZQpdRN/a3yQTwlePftnfQ2eZ0JARAP1IAcqnr64
0aKnLs3WzuI5gBTH2P2B9ix6DDD7uQINBFapILEBEAC4BpeMb7QNk6mfKk2nrkDF
dj2UgigAw2xsnkEUpHG6IufSlTOHF2hiJP7k86lbrIZGfQM9+9WBb2m+kik1Zh2I
53vvXD460ZtGBBzD7UMvAf5BOvrpnX7rmqtjLpGUvPhrQ/6h8LrBH29jCn/8C8yL
fl3B7A11C3YaxRVmR2TBXjMaYpmJ6Qhho4Jbw36/qscnZcPbFKTOs70uYAZD5hT7
PYBQs+496bLiDTjk5SkU2WsSRQG7+IDmTIMC0tzIPKcf1H14lXDAqSWDJsEuF/6g
zPc+Kg3GL0KW0hPHDb5+z5pfzKKBJQxWBwaPjAbGsn/WKFmuLMR6NzXZLoSt/EQq
Bd6Ud9goW+4JKeZhlVEWIZ/C+uNOr5eqEM1qiaEJW3Hrw5lxn4PEYZ2h59VPjM46
jsn5baRx54Wo/4oX8DpDlTyPM8ZOVkXDhHVgkHagygwLkiCQIyH9/htdVyIf0+33
zBS+oIsW8TJgbMaVrb9zy8BFcKfpICIjm5gCdxAb5xtO0pJSiSv4Ga0TTPwhzbHF
f6JpCohrg0ZpTpd2b4ZNOyshpWU8b9tdbSaoP3CQnpa7erzxOSI+xkAJODG+7DAz
qZrs0dMSAzlTG8gKmq5ceVWFA68gwDOZk6ObV4qcLaAakTYMWPzDMYjVD+NiNVJx
OYTBsj9lo8LaJSojNUmXzwARAQABiQIfBBgBAgAJBQJWqSCxAhsMAAoJEG32NKp2
CK8EKJwQAJAKqVuurLX2ApEgeLUVqb18s7kKmDC9MBy11zhmAzH51xrJimzg4j3v
QUjZqmV4iN4wPti/ME5RhwSgE9PeDXupsmGf//pD0YmTIWvOMhj4hASc4l6uNhlo
E2j5tN0A8IZBVQO1PvJdVYi6KJIYZy07qOg79qYQR4yYAXDLZQTlyBefvhVbk0H9
Ds8cC8gH9ag6Yn9t4TCfGhx7NP2j9W29OtnuDFt6GssgUt/1o1WILdMn2DzAdNr7
f6VDCSLKMjc3WQFe1XmrbR/xiH2SqKAOF6UIx++H4p7XPZyBmDcdbGzkputPYey0
tsvEN3ndyNLBsTgzPLALKiiXxvts798fjFWnZFVq1KmcZMj4+4yJLIBTBU1ZW0cC
F3e31qzAEDJmrKcwhN9IzVWAhRhHxpkKc2oADR5Lmq9CcXMOYZF9aS83YlIJKZ8U
WEgna802dzBcckBt2RvYxDYqs4iLLDdHjMGahTM3uieofIUXApUuSAMfTFvq5HSq
0/UU/UekE+NMUn3UtW2XLl9B49aB5bcUtOYtcbIJu8lhHuNXc2+zL/s9zCTsVx5P
XdCmEE81HfSxU/+7yhSxs10ixIporA2OxLYTUkzDDrqlK2N0ENjQp+39eVNTkGJd
15SnkvhqDukr11hTqdIgrwqrHTF6o2mVK79h2AqZqbi5ISCPc/MvuQINBFapIcYB
EACUE78/3F2br1jVCD8w/MC6rfxkleKjdfsafkkI10UPzhhMZAhgX1sXehF8luKC
sbWJ/d92j5dHOy5O8j6WuUfVWZgHQh3HqTBujz3lMYZXC3JCsUPajpQx5VH43JRj
pOz/Pw5Vy9RIS3UJ78vIAC2jqPc0wVknZmQ5JnF6nNGyU0AJYX7kwBM4685avPsI
tdpdix93Z3NEcqmS1B4PF3bCU96gHzAAw7IfCuF6TGzraHV4OPVJxa8GJp4Ziwnn
KD4vZzMx8FYMd8Egw50nsjFDL/DN2cFU512k/RSFFzXJ9y+NwnFwtZ3EobO8kQU0
fFqW4AFyfwsJKBAl8JlhgqzqeepRp1r0y7xmuxKjLxPgnps6ucSYbsqQhBztUCy0
fNW80gdb4kWz9bavY9165ALwCNuGQgLcLziKg4SwKiTJqurFcnAAgBg9X3pBFKJQ
seiarGELGaf/ptVA74auYNeqySJoSRpUe+/9WvX05hcvb786/KpOBBfYPtmWW2tZ
abo1T2FGiECNuqh0BqyxVQ3IVSBAY1IQjIGoXqR1Vh5kxmqW+5HxdlCKRZDBYk/5
ufT2d1GnLPkc+a8dm0PZhfzkM2fYblIpLEwd2w/xZZhmyYnDIkoFZxEu7RzT3t3y
ackos12ymyP4/qsCDf76VJ7KJhAE14MPiUHK0jGfalEZPQARAQABiQQ+BBgBAgAJ
BQJWqSHGAhsCAikJEG32NKp2CK8EwV0gBBkBAgAGBQJWqSHGAAoJEG2Jft9RigMd
GqEP/iIB3E2JpzlKAVBkBu0kQU7CHX7P4zcACayE3buOfzjgzLVk6IdwboH/LYT2
0w+Qwkqo1MV6uTe+831Hd9jRLyEuyxklGliYbXvdGbA+vtpdYcRiVnR61ATUg3Yu
d8MoLsqw+IK61W9e1M2puElKQ6Px/UmJTnfm3OsAnZ2BGJEpYJS1IYkxAKXqMVPE
bdZlMD+8/O9Aq0h5ySW0aIn6GNiUbzPzq9QMviMHR7Nolnw4aangtDUAmlqHO6Gh
3mUaIbPjt19HOFwHnCnv11CZRbyyoXonhZFxOATS13Av+hGg6J8S/gGSiV9fT+53
O1BodBygKJ8Y7JJDFIc/rTt05HaqHbNhucFqCUf5WuDOQabjiWWkgxjUdh050CzK
reYn8TGkTd6OCcmedUB3J4HmbhmwpkBw3ybAWvYhPoFi92/P7FtSThkaSbydnD8V
Men8wy+OuhJtvDnxLc4YGlxWqsW9WelsdxAZZNdSMPDrpmvkXJflmvCiP+4wehvx
Z3R2cgzOZjjYZlrD98IMjNdo87YTs4pxi+mEQNLYrLR404ZbhgTNkdFjKByM6EOl
F5xebblJjaTSLvaCs5p1lLdLTMSD3+1JUMLYRBuLN46ePLlxuDwtgum5hiNes2Ry
DGa/Gln73b/YtuGigcqi7ouesHuhcY0OwFuB4pu/8r+MbceYj3gP/jmZbVn2Yp3Q
qCgQYf/XFiZ5cgXitanBeYPa4A9WNpK2CuF0mtcwaE/vhTki37N9y5OQjpllJ/fL
equFTw7IPHcqcrQ7Fgeaf/lrWgVyNOUWiIJ9OJA4bjAJEMoD+qut8Ci8SPHe14Fr
94xP+ZrM7b9ZmEPvQTAQWip3Gtx3Ydv8U5z4eCeNPU6PnOxEEDvlBsyQp2wG+8ft
pLaidPZgfmdYoqLFIxhQiMUbQiDEmdZdlXCvMdOGF5oCxJDVFTovlaA7VgJMiUJG
hO8TVU+hsH6IeUMoOzCucKN6jlbaYTH9gOm0eclp+cE2BRSPYCh+B9J7uJm7uZR6
mW6uTx+pSyot6/eaJDo/EyBzvYdJK6eIdllFUfAO/S1LD160RPIsnevoYo7Yce3B
Ud2Pasrf+8Ptp/sFHT1hwMCz44SnkK4un9VQ4L0WsMnNQZKGmyyE1+Ydz6iHwKHf
lkGoJOmBU269MNKAh4qmwW9uG+T6Jxc50GLSzBAETtx3MYznEYjkcjCLnlb+x7Yh
HBRWMXiXCQ5atlWig2t/urrkrkSegVNnaP0fOSL71qihX0sRlsvEJCIWktKWVmzt
gcn75IpFzbTVTMtZoOgdFUSDPRSqNHA8hL+Itz2dUqHSktQ7pFK6ogIkvzx+7M7n
O6GuapL0x1I9SeIk9WX9VRrpZmENhlri
=JrM6
-----END PGP PUBLIC KEY BLOCK-----
```

Reply:
Updated.